added iframes lazyload

TrilipuT opened the pull request:
Also there are some refactoring.

Commit:
added gitignore
added iframes lazyload
refactoring

aFarkas commented:
Thanks, looks really great. 

It's funny, I'm not able to review ;-)

Some questions about the [`oembed_result` and the `embed_oembed_html`](https://github.com/TrilipuT/wp-lazysizes/blob/25a5db5cb4ba2a12b893a7397f0beb3756b0ef73/wp-lazysizes.php#L48-L49): 
- Do they always only return `iframe` elements or is there also the possibility that they will return `video` or `object` elements or something like this?
- I think [this code](https://github.com/TrilipuT/wp-lazysizes/blob/25a5db5cb4ba2a12b893a7397f0beb3756b0ef73/wp-lazysizes.php#L108) could break, if the iframe already has a class?

Something I would find great is, if the iframe markup could also be optionally wrapped in an responsive/intrinsic-ratio wrapper. My google search for "embed_oembed_html" revealed the following blogpost: http://diywpblog.com/embed-responsive-videos-with-wordpress/. What do you think?

Commit:
addClass method added

TrilipuT commented:
Yes, your'e right
- I have added an 'iframe' checking before adding 'lazyload' class
- I have added method to add class into html. It also contain check for existing classes. Should work fine now.

I have created develop branch. Lets check it and then will make merge into master. I will close this pull request due to it's oldness

aFarkas commented:
@TrilipuT 
Just wanted to repeat: I ow you. Just in case you missed it: you can prepend your signature as author.
